andela/activo-web
# #158869814 Move images and fonts to Cloudinary

**What does this PR do?**
Moves images and fonts to Cloudinary and update the links.

**Description of Task to be completed?**
- move images to Cloudinary
- move fonts to Cloudinary
- update image links to reflect Cloudinary URLs
- update font links to reflect Cloudinary URLs

**How should this be manually tested?**
After cloning or fetching branch, run the application. Fonts should be correctly displayed, and image assets should load properly.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#158869814](https://www.pivotaltracker.com/story/show/158869814)

**Screenshots (if appropriate)**

<img width="1280" alt="screen shot 2018-07-12 at 2 24 48 pm" src="https://user-images.githubusercontent.com/24798364/42635946-629d2568-85df-11e8-90ff-8a93173cdf6f.png">

**Questions:**

N/A
